packages/frontend2/src/app/(legacy)/scaling-next/projects/[slug]/tvl-breakdown/_components/tables/natively-minted-table.tsx

```diff
@@ -0,0 +1,41 @@
+'use client'
+import { getCoreRowModel } from '@tanstack/react-table'
+import { TokenTable } from '~/app/_components/table/token-breakdown-table'
+import { useTable } from '~/hooks/use-table'
+import { type ExtendedProjectTvlBreakdown } from '../../_utils/assign-token-meta-to-breakdown'
+import { nativelyMintedColumns } from './columns/natively-minted-columns'
+import { TableSum } from './table-sum'
+
+export type NativelyMintedTokenEntry =
+  ExtendedProjectTvlBreakdown['breakdown']['native'][number]
+
+interface Props {
+  tokens: NativelyMintedTokenEntry[]
+}
+
+export function NativelyMintedTable(props: Props) {
+  const sum = props.tokens.reduce((acc, token) => {
+    return acc + Number(token.usdValue)
+  }, 0)
+
+  const table = useTable({
+    data: props.tokens,
+    initialState: {},
```

Comment:
no need to pass then

Reply:
https://github.com/l2beat/l2beat/pull/4855/commits/7aeacd13710eff90f521dec2fdbc2f3147025e7a